### PR TITLE
Rubicion: Update userExt if eids resolved

### PR DESCRIPTION
**Pull request by SerhiiNahornyi**

### CODE REVIEW

File: src/test/java/org/prebid/server/bidder/rubicon/RubiconBidderTest.java [hunk 1]
```diff
@@ -1623,6 +1623,43 @@ public void makeHttpRequestsShouldNotCreateUserExtTpIdWithAdServerEidSourceIfExt
                         .build());
     }
 
+    @Test
+    public void makeHttpRequestsShouldCreateUserExtEidsithAdServerEidSource() {
```
Comment (Net-burst) [inline src/test/java/org/prebid/server/bidder/rubicon/RubiconBidderTest.java:1627]: Typo in test name: `ExtEidsith` should be `ExtEidsWith`

**Review by Net-burst — APPROVED** [review]
LGTM with remark

File: src/test/java/org/prebid/server/bidder/rubicon/RubiconBidderTest.java (hunk 1 above)
Comment (SerhiiNahornyi) [reply src/test/java/org/prebid/server/bidder/rubicon/RubiconBidderTest.java]: Nice catch!

**Review by Net-burst — APPROVED** [review]
LGTM

**Pull request merged**